src/Command/ModelCommand.php

```diff
@@ -475,9 +475,9 @@ public function findBelongsToMany(\Cake\ORM\Table $model, array $associations):
      *
      * @param \Cake\ORM\Table $model The model to introspect.
      * @param \Cake\Console\Arguments $args CLI Arguments
-     * @return string
+     * @return string|null
      */
-    public function getDisplayField(\Cake\ORM\Table $model, \Cake\Console\Arguments $args): string
+    public function getDisplayField(\Cake\ORM\Table $model, \Cake\Console\Arguments $args): ?string
```

Comment:
Return value for core's `Table:getDisplayField()` is `string|string[]` so adding return typehints of `string` or `?string` both would be incorrect here.

Though we should ensure that wherever the return values of this method is consumed handles array values properly too.